components/appointments/id/index.tsx

```diff
@@ -66,44 +70,91 @@ export default function Appointment({ aid, session }: AppointmentProps) {
-              <CellRenderer
-                icon="solar:calendar-bold-duotone"
-                value={format(new Date(appointment.date), 'EEEE, MMM d, yyyy ')}
-                classNames={{ icon: 'text-gray-500 bg-gray-100' }}
-                iconSize={18}
-                className="items-center rounded-lg bg-gray-100 px-2 py-1 text-xs text-gray-500"
-              />
-              <CellRenderer
-                icon="mdi:clock-outline"
-                value={format(new Date(appointment.date), 'h:mm a')}
-                classNames={{ icon: 'text-gray-500 bg-gray-100' }}
-                iconSize={18}
-                className="items-center rounded-lg bg-gray-100 px-2 py-1 text-xs text-gray-500"
-              />
-            </div>
+    <div className="container mx-auto p-8">
+      <nav className="mb-6 flex flex-col items-start justify-between gap-4 md:flex-row md:items-center">
+        <div>
+          <div className="flex items-center gap-2">
+            <h1 className="text-xl">Appointment Details</h1>
+            <p>
+              <span>#{appointment.aid}</span>
+            </p>
           </div>
-        </nav>
-
-        <div className="grid gap-6 md:grid-cols-2">
-          <Card className="w-full">
-            <CardHeader>
-              <h2 className="text-default-500">Pateint Details</h2>
+          <div className="flex items-center gap-2">
+            {renderChip({ item: appointment.status })}
+            <CellRenderer
+              icon="solar:calendar-bold-duotone"
+              value={format(new Date(appointment.date), 'EEEE, MMM d, yyyy ')}
+              classNames={{ icon: 'text-gray-500 bg-gray-100' }}
+              iconSize={18}
+              className="rounded-lg bg-gray-100 px-2 py-1"
+            />
+            <CellRenderer
+              icon="solar:clock-circle-linear"
+              value={format(new Date(appointment.date), 'h:mm a')}
+              classNames={{ icon: 'text-gray-500 bg-gray-100' }}
+              iconSize={18}
+              className="rounded-lg bg-gray-100 px-2 py-1"
+            />
+            <CellRenderer
+              icon="solar:tag-bold"
+              iconSize={18}
+              value={appointment.type}
+              classNames={{
+                icon: 'text-primary bg-primary-50',
+                value: 'text-primary',
+              }}
+              className="rounded-lg bg-gray-100 px-2 py-1"
+            />
+          </div>
+        </div>
+        <div className="flex items-center">
+          <Button
+            color="primary"
+            variant="light"
+            startContent={<Icon icon="solar:printer-outline" width="18" />}
+          >
+            Print
+          </Button>
+          <Button
+            color="primary"
+            variant="light"
+            startContent={<Icon icon="solar:share-bold" width="18" />}
+          >
+            Share
+          </Button>
+          <Dropdown>
```

Comment:
change placement to bottom-right
```suggestion
          <Dropdown placement="bottom-right" >
```